sql/ddl_internal.sql

```diff
@@ -571,6 +448,25 @@ BEGIN
 END
 $BODY$;
 
+CREATE OR REPLACE FUNCTION _timescaledb_internal.add_dimension(
+    main_table               REGCLASS,
+    column_name              NAME,
+    num_slices               INTEGER = NULL,
+    interval_length          anyelement = NULL::BIGINT,
```

Comment:
Should we rename this parameter to `chunk_time_interval` to be consistent with `create_hypertable()` and `set_chunk_time_interval()`?

Reply:
yes